baselines/dom.generated.d.ts

```diff
@@ -3465,6 +3466,9 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     readonly scrollWidth: number;
     readonly tagName: string;
     innerHTML: string;
+    readonly assignedSlot: HTMLSlotElement | null;
```

Comment:
The property is optional on the spec, we should make it optional here so that `undefined` can be accepted as well.